src/Setup_geometry.jl

```diff
@@ -593,7 +593,7 @@ end
 
 
 """
-        add_polygon!(Phase, Temp, Grid::AbstractGeneralGrid; xlim::Vector(), ylim=Vector(2), zlim=Vector(), phase = ConstantPhase(1), T=nothing, cell=false )   
+        add_polygon!(Phase, Temp, Grid::AbstractGeneralGrid; xlim=Tuple{}, ylim=Tuple{2}, zlim=Tuple{}, phase = ConstantPhase(1), T=nothing, cell=false )   
```

Comment:
What is the point of `Tuple{}` and `Tuple{2}`? If you want to make empty tuples you can to just `xlim=()`  

```julia
julia> Tuple{} == ()
false
```